devprocess.tex

```diff
@@ -0,0 +1,7 @@
+\section{Development Process} \label{sect:devproc}
+
+DM is essentially a large software project, more we are developing scientific software with the in uncertainties that brings with it. 
+An agile \citep{it:agile} is particularly suited to scientific software development.  The development follows a six month  cyclical approach and  DM  products are under continuous
+integration using the application software Jenkins. All code is developed in the GitHub open source repository under an open source license.
```

Comment:
Are we being deliberately cagey with the license? We use GPLv3 and for documentation we use creative commons for the web docs and DMTNs (I'm not entirely sure what our license is for LDMs).